CHANGELOG.md

```diff
@@ -1 +1 @@
-packages/ember-auto-import/CHANGELOG.md
\ No newline at end of file
+[packages/ember-auto-import/CHANGELOG.md](https://github.com/embroider-build/ember-auto-import/blob/main/packages/ember-auto-import/CHANGELOG.md)
```

Comment:
this should be a relative link
current pr ties it to github.com so it's not usable on a checked out repo to jump to file, as well as always pointing to the main branch which I'm not convinced is always the right thing to be doing